app/code/core/Mage/Eav/Model/Entity/Attribute/Option.php

```diff
@@ -34,4 +34,19 @@ public function _construct()
     {
         $this->_init('eav/entity_attribute_option');
     }
+
+    /**
+     * Retrieve swatch value
+     *
+     * @return Mage_Eav_Model_Entity_Attribute_Option_Swatch|bool
+     */
+    public function getSwatch()
```

Comment:
```suggestion
    public function getSwatchValue()
```

not sure about this but since it returns the value... and in this case we can remove the `* Retrieve swatch value` comment

Reply:
i've applied your suggestions but keep comment as "Retrieve swatch hex value"